packages/paste-core/components/checkbox/__tests__/checkbox.test.tsx

```diff
@@ -224,6 +238,66 @@ describe('Checkbox event handlers', () => {
   });
 });
 
+describe('Customization', () => {
+  it('Should set an element data attribute for Checkbox', (): void => {
```

Comment:
FWIW you don't need to type the assertion callback in an `it` statement.

🪙 🪙 Might be good to remove, since it adds unnecessary noise.

Reply:
ah i see! hmm I'd seen in it one of Rich's PR's a while back and added in bc of that, but I can remove these easily. Any other opinions on the typing here?